plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/ast/SilverCallable.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.viper.ast
+
+import scala.collection.Seq
+import viper.silver.ast.Exp
+
+/**
+ * Interface used to wrap Viper nodes with an argument list (such as function applications, method calls, ...).
+ */
+interface IntoSilverCallable {
```

Comment:
This is the only place we use `Silver` like this, as the name for a wrapper.  Would just calling it a `Callable` cause issues?